src/libmongoc/tests/test-mongoc-client.c

```diff
@@ -3416,12 +3445,12 @@ test_client_appname (bool pooled, bool use_uri)
 
    mock_server_replies_simple (request, server_reply);
    if (!pooled) {
-      _respond_to_ping (future, server);
+       _respond_to_ping (future, server, false); // JFW: mongoc_client_uses_server_api(client));
```

Comment:
Running the test locally suggests the 3rd argument should be `true`. `mongoc_client_uses_server_api(client)` returns `false`.